config-model-api/src/main/java/com/yahoo/config/application/api/DeploymentSpec.java

```diff
@@ -168,17 +198,21 @@ public static DeploymentSpec fromXml(String xmlForm) {
 
             if (environment == Environment.prod) {
                 for (Element stepTag : XML.getChildren(environmentTag)) {
-                    if (stepTag.getTagName().equals("delay"))
-                        steps.add(new Delay(Duration.ofSeconds(longAttribute("hours",   stepTag) * 60 * 60 +
-                                                               longAttribute("minutes", stepTag) * 60 +
-                                                               longAttribute("seconds", stepTag))));
-                    else // a region: deploy step
-                        steps.add(new DeclaredZone(environment,
-                                                   Optional.of(RegionName.from(XML.getValue(stepTag).trim())),
-                                                   readActive(stepTag)));
+                    if (stepTag.getTagName().equals("delay")) {
+                        steps.add(new Delay(Duration.ofSeconds(longAttribute("hours", stepTag) * 60 * 60 +
+                                                                       longAttribute("minutes", stepTag) * 60 +
```

Comment:
You messed up the alignment here

Reply:
Thanks, I think it's due to IntellIJ's default behavior. I'll restore it in my next PR.